package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@saucelabs/cypress-junit-plugin",
+  "description": "Sauce Cypress JUnit Plugin",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "npx rimraf lib && tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "lint": "prettier --check '**/*.{js,ts,mjs,cjs}' && eslint .",
+    "fmt": "prettier --write '**/*.{js,ts,mjs,cjs}'",
+    "release": "tsc && release-it --github.release",
+    "release:ci": "tsc && npm run release -- --ci --no-git.requireCleanWorkingDir",
+    "release:patch": "npm run release -- patch",
+    "release:minor": "npm run release -- minor",
+    "release:major": "npm run release -- major"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/saucelabs/cypress-junit-plugin.git"
+  },
+  "bugs": {
+    "url": "https://github.com/saucelabs/cypress-junit-plugin/issues"
+  },
+  "author": "devx <devx@saucelabs.com>",
+  "files": [
+    "lib"
+  ],
+  "license": "MIT",
+  "engines": {
+    "node": ">=16.13.2"
+  },
+  "dependencies": {
+    "debug": "^4.3.4",
+    "fast-xml-parser": "^4.3.4",
+    "xml-escape": "^1.1.0",
+    "xmlbuilder2": "^3.1.1"
+  },
+  "peerDependencies": {
+    "cypress": ">=13"
```

Comment:
The README mentions cypress 10 as the minimum. Should that be reflected here in the peer dep?

Reply:
hmm.. Mention it as`Cypress 10+` because there were breaking changes between Cypress 9 and Cypress 10. So the README is trying to indicate that it's supporting the latest Cypress config.
I also checked the similar dependencies setting in `sauce-cypress-plugin`: https://github.com/saucelabs/sauce-cypress-plugin/blob/main/package.json#L45. And it's `>= 13`.

Reply:
sauce-cypress-plugin only supports cypress >= 13 (see https://github.com/saucelabs/sauce-cypress-plugin/pull/55), so the peer dep is correct there. If the api your plugin is accessing is in fact supported back to cypress 10, then that should be reflected here in the peer dep. It's not going to break anything if you leave it, but it can generate unnecessary warnings when this plugin is installed, which wouldn't be a good starting experience.

Reply:
If then, I'd say `sauce-cypress-plugin` has an accurate README actually.

Reply:
Updated README to indicate it's actually supporting cypress 13+